components/stacklayout/overview.md

```diff
@@ -162,12 +134,11 @@ Sometimes you need to create a more complex layout that includes both horizontal
 </style>
 ````
 
->caption The result from the code snippet above
+## Next Steps
 
-![Nested StackLayouts can create complex layout](images/stacklayout-overview-nested-stacklayouts-example.png)
+* [Configure Layout]({%slug stacklayout-layout%})
```

Comment:
Use more keywords in links. In this case, "layout" will be repeated, so we can use something else.

```suggestion
* [Configure StackLayout orientation, spacing and alignment]({%slug stacklayout-layout%})
```